README.md

```diff
@@ -1,41 +1,82 @@
-# Website
+# Scaffold-ETH 2 Docs
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This website contains Scaffold-ETH 2 Documentation and is built using [Docusaurus 2](https://docusaurus.io/).
 
-### Installation
+## Requirements
 
-```
-$ yarn
-```
+To run the Documentation site locally, but first ensure you have [Node](https://nodejs.org/en/download/), [Yarn](https://yarnpkg.com/) and [Git](https://git-scm.com/downloads) installed.
 
-### Local Development
+## Installation
+
+Clone the repository, install all dependencies as well as build all local packages, and then start Docusaurus.
 
 ```
-$ yarn start
+git clone https://github.com/scaffold-eth/se2-docs.git
+cd se2-docs
+yarn install
+yarn start
```

Comment:
We are using `npm` as our package manager for this, we should migrate to `yarn` in future

Lets update this for now to : 

```shell
npm install

npm run start
```

Reply:
Ups! Saw it in the old Readme and always used it, so never questioned hah 

Now I've realized there is no yarn.lock file in the repo, updating! 👍